### PR TITLE
chore(sci-fi): add rooms to first floor

### PR DESCRIPTION
**Pull request by Manas-Tiwari**
extended B room and made some other rooms so that sci-fi dungeon story looks a little bit more deep and intresting.

### CODE REVIEW

File: english/sci-fi-dungeon/1/the-end.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+**You fought with pride.**
+
+You are dead ☠️.
+
+**THE END 🎬**
+
+[Restart the game](../begin-journey.md)
```
Comment (ming-tsai) [inline english/sci-fi-dungeon/1/the-end.md:7]: 1. A dungeon has a lot of ending floor, we could not create a step that its name is `the end`, due to depending on the story we could have different `the end`.
2. If doesn't have another room reference for this file, so please delete this file.

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @Manas-Tiwari,
Some suggestions

File: english/sci-fi-dungeon/1/the-end.md (hunk 1 above)
Comment (Manas-Tiwari) [reply english/sci-fi-dungeon/1/the-end.md:7]: yes sure. no problem. 😄

Comment (Manas-Tiwari) [reply english/sci-fi-dungeon/1/the-end.md]: hi ming, can you do a little favor on me by adding hacktoberfest-accepted tag on this PR, please as i have to complete 4 PRs this month.
Tysm.

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**